cms/src/components/content/GroupSelector.vue

```diff
@@ -0,0 +1,147 @@
+<script setup lang="ts">
+import { computed, ref, toRaw, toRefs } from "vue";
+import { ChevronUpDownIcon } from "@heroicons/vue/20/solid";
+import {
+    Combobox,
+    ComboboxButton,
+    ComboboxInput,
+    ComboboxLabel,
+    ComboboxOption,
+    ComboboxOptions,
+} from "@headlessui/vue";
+import type { Group } from "@/types"; // Assuming you have a Group type defined
+import LTag from "./LTag.vue"; // Assuming you have a component for rendering groups
```

Comment:
Not sure where these comments come from? Copilot? 😄

Reply:
😄😄